test/script.js

```diff
@@ -142,4 +142,19 @@ describe('Script', function () {
     should(thrownException).not.equal(null);
     thrownException.message.should.equal('Script is destroyed');
   }));
+
+  it('should support custom log handler', co.wrap(function *() {
+    const script = yield session.createScript(
+      '"use strict";' +
+      '' +
+      'console.error(new Error("test message"))');
+
+    script.setLogHandler(function(level, text) {
+      should(level).equal('error');
+      should(text).equal('Error: test message');
+    });
+
+    yield script.load();
+    should('a').equal('a');
```

Comment:
Instead of this placeholder line it would be good to wait for the log handler to get called. Perhaps the handler could resolve a `Promise` that we `yield` here.

Reply:
Oh sorry, I forgot to delete that